src/mappings/entities/mapSettlement.ts

```diff
@@ -24,7 +24,6 @@ const updateEvents: EventIdEnum[] = [
   EventIdEnum.InstructionAuthorized,
   EventIdEnum.InstructionUnauthorized,
   EventIdEnum.InstructionAffirmed,
-  EventIdEnum.InstructionFailed,
```

Comment:
InstructionFailed isn't an extrinsic, but something the gets triggered and doesn't have a signer.

To prevent the block, since handleInstructionUpdate is only adding the signers, its save to0 exclude from there. Its being handled in the finalizedEvents